src/components/ComboBox/MultiComboBox.tsx

```diff
@@ -327,6 +333,18 @@ const ActualMultiComboBox = <T,>(
       } else if (e.key === 'Right' || e.key === 'ArrowRight') {
         e.stopPropagation()
         focusNextDeletionButton()
+      } else if (
+        e.key === 'Backspace' &&
+        !inputValue &&
+        selectedItems.length > 0 &&
+        selectedItems[selectedItems.length - 1].deletable !== false
```

Comment:
以下でも良い？と思ったけど、そもそもdeletableはoptionalなのか
```suggestion
        selectedItems[selectedItems.length - 1].deletable
```

Reply:
!!selectedItems[selectedItems.length - 1].deletable
ならありか